services/frontend-service/pkg/cmd/server.go

```diff
@@ -52,6 +70,7 @@ type Config struct {
 	AzureClientId       string `default:"" split_words:"true"`
 	AzureTenantId       string `default:"" split_words:"true"`
 	AzureRedirectUrl    string `default:"" split_words:"true"`
+	Version             string `default:""`
```

Comment:
here too

Reply:
It is fixed now

Reply:
We will keep it as "Version" here because it will be in helm "KUBERPULT_VERSION", because we adding the "KUERPULT" prefix by default, if we make it "KuberpultVersion" it will be "KUBERPULT_KUBERPULT_VERSION"
but in all other places, it will be "KuberpultVersion"